test/libsolidity/syntaxTests/controlFlow/storageReturn/while_err.sol

```diff
@@ -8,4 +8,4 @@ contract C {
     }
 }
 // ----
-// TypeError: (87-98): This variable is of storage pointer type and might be returned without assignment and could be used uninitialized. Assign the variable (potentially from itself) to fix this error.
+// TypeError: (87-98): This variable is of storage pointer type and is accessed without prior assignment.
```

Comment:
In this case, the error is misleading. It should say "is returned without prior assignment", or even "can be returned" (in case you have multiple branches).